server/app/views/v1/host_nodes/health.json.jbuilder

```diff
@@ -3,8 +3,14 @@ json.name @node.name
 json.node_number @node.node_number
 json.initial_member @node.initial_member?
 
+json.status @node.status
+json.connected_at @node.connected_at
+json.disconnected_at @node.disconnected_at
 json.connected @node.connected
+
 json.etcd_health do
-  json.health @etcd_health[:health]
-  json.error @etcd_health[:error]
+  json.health @node_health.dig(:etcd_health, :health)
+  json.error @node_health.dig(:etcd_health, :error)
```

Comment:
Changed this yet again to avoid the `@node_health` vs `@node_errors` duality - the `health.json` template is only used if the server is able to query the node health.

Offline nodes and RPC timeouts how result in HTTP 422 responses with a standard `{error: ...}` hash from the `GET /v1/nodes/:id/health` API.